prototype/docs/southbound/device-factory/core.md

```diff
@@ -0,0 +1,217 @@
+
+*Note that depending on the parser implementation, the available entries might not match exactly the payload content.*
+
+### Mapping key
+
+The key of the mapping entry can be in the following formats:
+* `@xxx`: a placeholder used by the device factory to describe the update and to find the provider to update. Here are the placeholders sensiNact defines:
+  * Provider description
+    * `@model`: the name of the model of the provider to update
+    * `@provider`: the name of the provider to update. **This placeholder is mandatory**.
+    * `@name`: the friendly name of the provider (`admin/friendlyName`)
+  * Location
+    * `@location`: the location of the provider (`admin/location`). The content is expected to be either a GeoJSON object or a short location string (`latitude:longitude[:altitude]`). This placeholder overrides `@latitude`, `@longitude` and `@altitude` if they are defined.
+    * `@latitude`: the latitude of the provider, to construct its location. This placeholder **must** be combined with `@longitude`.
+    * `@longitude`: the longitude of the provider, to construct its location. This placeholder **must** be combined with `@latitude`.
+    * `@altitude`: the altitude of the provider, to construct its location. This placeholder **must** be combined with `@latitude` and `@longitude`.
+  * Update timestamp
+    * `@timestamp`: the timestamp of the update either in millisecond or second precision (auto-detected). Overrides `datetime`, `@date` and `@time` if they are defined.
+    * `@datetime`: the date and time of the update. Overrides `@date` and `@time` if they are defined. The datetime format can be defined in the mapping options.
+    * `@date`: the date of the update, should be used with `@time`. The date format can be defined in the mapping options.
+    * `@time`: the time of the update, should be used with `@date`. The time format can be defined in the mapping options.
+* `$xxx`: definition of a variable that can be reused in or as a mapping value. The variable can be used in other mapping key or in record paths using the `${xxx}` syntax.
+* `"svc/rc"`: the parser value will be stored in a the resource `rc` of the service `svc`
+
+### Mapping value
+
+The value of a mapping can be defined in many different formats.
+
+In the examples of this section, we consider the input payload being a CSV file with the following content:
+
+```csv
+sensor,serial,nbVehicles,nbPersons
+A1,ABC12,6,2
+B1,ABC13,0,10
+```
+
+#### Record path
+
+A mapping defined by a record path will get the value from the parser "as is".
+This means the resource value will contain it in its parsed type.
+For example, all the values from the CSV parser are strings, whereas the JSON parser will detect strings, numbers, booleans, arrays, ...
+
+Here is an example of mapping the value of the resource `serial` in the service `sensor` from the record path `serial`:
+```json
+{
+    "sensor/serial": "serial"
+}
+```
```

Comment:
This mapping should include a `@provider` and should say what the mapping results are for the example input.